scripts/scylla-get.sh

```diff
@@ -0,0 +1,193 @@
+    echo "Installing Scylla release $SCYLLA_VERSION for RHEL ..."
+    rhel_install
+  elif [ "$ID" = "ol" ]; then
+    echo "Installing Scylla release $SCYLLA_VERSION for Oracle Linux ..."
+    oraclelinux_install
+  elif [ "$ID" = "debian" ]; then
+    echo "Installing Scylla release $SCYLLA_VERSION for Debian ..."
+    debian_install
+  elif [ "$ID" = "ubuntu" ]; then
+    echo "Installing Scylla release $SCYLLA_VERSION for Ubuntu ..."
+    ubuntu_install
+  else
+    echo "Operating system '$ID' is not supported by this installer."
+    exit 1
+  fi
+
+ echo "Scylla installation done!"
+}
+
+usage() {
+  cat 1>&2 <<EOF
+scylla-get
+The platform agnostic installer for Scylla.
+
+USAGE:
+    scylla-get [FLAGS] [OPTIONS]
+
+FLAGS:
+    -h, --help              Prints help information
+
+OPTIONS:
+	--scylla-version <version>                 Scylla version to install (default: $SCYLLA_VERSION)
+EOF
+}
+
+require_cmd() {
+  CMD=$1
+  if ! command -v "$CMD" &> /dev/null
+  then
+    echo "Please make sure '$CMD' is installed on this machine."
+    exit
+  fi
+}
+
+centos_install() {
+  if [[ "$VERSION_ID" = "7" || "$VERSION_ID" = "8" ]]; then
+    yum install --assumeyes --quiet epel-release
+    curl -s -L -o /etc/yum.repos.d/scylla.repo http://downloads.scylladb.com/rpm/centos/scylla-$SCYLLA_RELEASE.repo
+    yum install --assumeyes --quiet scylla
+  else
+    echo "CentOS $VERSION_ID is not supported by this installer."
+    exit 1
+  fi
+}
+
+rhel_install() {
+  if [[ "$VERSION_ID" = "8.2" ]]; then
+    yum install --assumeyes --quiet https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+    curl -s -L -o /etc/yum.repos.d/scylla.repo http://downloads.scylladb.com/rpm/centos/scylla-$SCYLLA_RELEASE.repo
+    yum install --assumeyes --quiet scylla
+  else
+    echo "RHEL $VERSION_ID is not supported by this installer."
+    exit 1
+  fi
+}
+
+oraclelinux_install() {
+  if [[ "$VERSION_ID" = "8.2" ]]; then
+    yum install --assumeyes --quiet https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+    curl -s -L -o /etc/yum.repos.d/scylla.repo http://downloads.scylladb.com/rpm/centos/scylla-$SCYLLA_RELEASE.repo
+    yum install --assumeyes --quiet scylla
+  else
+    echo "Oracle Linux $VERSION_ID is not supported by this installer."
+    exit 1
+  fi
+}
+
+debian_install() {
+  SCYLLA_DEBIAN_URL_PREFIX="http://downloads.scylladb.com/deb/debian"
+  if [ "$VERSION_ID" = "9" ]; then
+    apt update && apt-get install -qq apt-transport-https curl gnupg2 dirmngr
+    apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 5e08fbd8b5d6ec9c
+    curl -s -L -o /etc/apt/sources.list.d/scylla.list "$SCYLLA_DEBIAN_URL_PREFIX/scylla-$SCYLLA_RELEASE-stretch.list"
+    apt update && apt-get install -qq scylla
+  elif [ "$VERSION_ID" = "10" ]; then
+    apt update && apt-get -qq install apt-transport-https curl gnupg2 dirmngr
+    apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 5e08fbd8b5d6ec9c
```

Comment:
Why are the keys different for 9 and 10?

Reply:
Will check.

Reply:
They keys were the _same_. It's apparent now after code consolidation.